website/docs/reference/dbt-jinja-functions/run_query.md

```diff
@@ -95,3 +95,15 @@ You can also use `run_query` to perform SQL queries that aren't select statement
 ```
 
 </File>
+
+
+If you wish to check whether `run_query` returned any rows or not you can use `length` filter. 
```

Comment:
```suggestion
To verify whether `run_query` returned any rows or not &mdash; use the `length` filter.  Wrap the logic in an [if execute]([/reference/dbt-jinja-functions/execute) block to avoid unexpected behavior during parsing. 
```